cogs/hideout/boost_roles.py

```diff
@@ -0,0 +1,205 @@
+        icon: Optional[discord.Attachment] = None,
+        emoji: Optional[str] = None,
+    ):
+        """Edits your boost role."""
+        assert isinstance(interaction.user, discord.Member) and interaction.guild
+
+        if BOOSTER_ROLE_ID not in [role.id for role in interaction.user.roles]:
+            return await interaction.response.send_message("You're currently not boosting the server.", ephemeral=True)
+
+        colour_: discord.Colour = discord.Colour.default()
+        icon_ = ""
+
+        db = await interaction.client.pool.fetchrow("SELECT * FROM booster_roles WHERE user_id = $1", interaction.user.id)
+        if db is None:
+            return await interaction.response.send_message(
+                "You don't have a booster role, create one with `/boost role create`.", ephemeral=True
+            )
+
+        if not any((name, colour, icon, emoji)):
+            return await interaction.response.send_message("You need to provide at least one argument.", ephemeral=True)
+
+        if colour is not None:
+            with suppress(ValueError):
+                colour_ = discord.Colour.from_str(colour)
+
+        if icon and emoji:
+            return await interaction.response.send_message("You can not supply both `icon` and `emoji`.", ephemeral=True)
+
+        if emoji is not None:
+            conv = await self.get_emoji(emoji)
+            if conv is None:
+                return await interaction.response.send_message("Could not parse that emoji.", ephemeral=True)
+
+            icon_ = conv
+
+        elif icon is not None:
+            if icon.size > 256 * 10**3:  # 256 kB
+                return await interaction.response.send_message("The icon size needs to be less than 256kB.", ephemeral=True)
+
+            icon_ = await icon.read()
+
+        role = interaction.guild.get_role(db["role_id"])
+        assert role
+
+        role = await role.edit(
+            name=name or MISSING,
+            colour=colour_ if colour_ != discord.Colour.default() else MISSING,
+            display_icon=icon_ or MISSING,
+        )
+
+        assert role
+
+        query = """
+            UPDATE booster_roles
+                SET role_name = $1,
+                    role_colour = $2,
+                    role_icon = $3
+                WHERE role_id = $4
+            RETURNING *
+        """
+
+        db = await interaction.client.pool.fetchrow(
+            query, role.name, str(role.colour), role.icon and await role.icon.read(), role.id
+        )
+
+        assert db
+
+        embed: Optional[discord.Embed] = MISSING
+        role_icon: Optional[discord.File] = MISSING
+        if role.icon is not None:
+            role_icon = discord.File(filename=f"icon.png", fp=BytesIO(db["role_icon"]))
+            embed = discord.Embed().set_thumbnail(url="attachment://icon.png")
+
+        text = f"""
+            Successfully edited {role.mention}
+            Name: {role}
+            Color: {db['role_colour'].upper()}
+        """
```

Comment:
```suggestion
        text = (
            f"Successfully edited {role.mention}"
            f"\nName: {role}"
            f"\nColor: {db['role_colour'].upper()}"
        )
```
Doing it this way prevents unwanted indentation showing up on discord